vendor/vendor.json

```diff
@@ -1094,6 +1094,12 @@
 			"revision": "ffef3ffc77bec026fefa6f76bd53d158cfa0e669",
 			"revisionTime": "2017-03-24T09:53:51Z"
 		},
+		{
+			"checksumSHA1": "BfjvAbPn5t/2KrdC5E7418OuzV8=",
+			"path": "github.com/gofrs/uuid",
```

Comment:
Can you checkout from a tag like `govendor fetch github.com/gofrs/uuid@v3.1.1`?

Reply:
we should probably to the same in go-ucfg so we use the exact same version.